specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/examples/AutoDismissAlertsRules/GetAutoDismissAlertsRulesWithAlertType_example.json

```diff
@@ -0,0 +1,50 @@
+{
+  "parameters": {
+    "api-version": "2019-01-01-preview",
+    "subscriptionId": "20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+    "alertType": "IpAnomaly"
+  },
+  "responses": {
+    "200": {
```

Comment:
>200 [](start = 5, length = 3)

200 or 204?

Relevant also for other examples

Reply:
(I just saw some place uses 204. Not sure what each means and if there is any place with wrong value...)

---
In reply to: [351815292](https://github.com/yibirnba/azure-rest-api-specs/pull/5#discussion_r351815292) [](ancestors = 351815292)

Reply:
204 it is ok with no content. here we return content